listenbrainz/webserver/static/js/src/components/Pill.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from "react";
+
+type PillProps = {
+  active?: boolean;
+  type?: "primary" | "secondary";
+  style?: React.CSSProperties;
+  className?: string;
+  onClick?: React.MouseEventHandler<HTMLButtonElement>;
+  [key: string]: any;
+};
+
+export default class Pill extends React.Component<PillProps> {
```

Comment:
i feel like this could be a function as well?

Reply:
Yep will do.